Dockerfile

```diff
@@ -12,6 +12,15 @@ RUN git clone http://git.ipxe.org/ipxe.git && \
       sed -i -e "s/#undef.*NET_PROTO_IPV6/#define NET_PROTO_IPV6/g" config/general.h && \
       make bin/undionly.kpxe bin-x86_64-efi/ipxe.efi bin-x86_64-efi/snponly.efi
 
+RUN dd bs=1024 count=2880 if=/dev/zero of=esp.img && \
```

Comment:
May be a future-proof way would be to measure the sizes of the files we want to write onto our floppy, and size the floppy accordingly?

Reply:
I agree, but we don't have the time to be elaborate right now.

Reply:
Adding a note to remind us to do it later.